components/combobox/stories/combobox.stories.js

```diff
@@ -165,6 +170,21 @@ QuietGroup.parameters = {
 	chromatic: { disableSnapshot: true },
 };
 
+/**
+ * Comboboxes have a read-only option for when content in the disabled state still needs to be shown. This allows for content to be copied, but not interacted with or changed. A combobox does not have a read-only option if no selection has been made. To enable this feature, add the `.isReadOnly` class to the combobox. To enable this feature, add the .isReadOnly class to the combobox. Then within the nested textfield component, add the .isReadOnly class and readonly attribute to the <input> element.
```

Comment:
```suggestion
 * Comboboxes have a read-only option for when content in the disabled state still needs to be shown. This allows for content to be copied, but not interacted with or changed. A combobox does not have a read-only option if no selection has been made. To enable this feature, add the `.isReadOnly` class to the combobox. To enable this feature, add the .isReadOnly class to the combobox. Then within the nested textfield component, add the .isReadOnly class and readonly attribute to the `<input>` element.
```
One last update and this should be good to go! This was rendering an input on the Docs.

Reply:
@jawinn Good catch!